src/Model/Filter/Like.php

```diff
@@ -29,31 +54,49 @@ public function process()
             return;
         }
 
+        $comparison = $this->config('comparison');
+        $valueMode = $this->config('valueMode');
+        $value = $this->value();
+        $isMultiValue = is_array($value);
+
         $conditions = [];
         foreach ($this->fields() as $field) {
-            $left = $field . ' ' . $this->config('comparison');
-            $right = $this->_wildcards($this->value());
-
-            $conditions[] = [$left => $right];
+            $left = $field . ' ' . $comparison;
+            if ($isMultiValue) {
+                $valueConditions = [];
+                foreach ($value as $val) {
+                    $right = $this->_wildcards($val);
+                    if ($right !== false) {
+                        $valueConditions[] = [$left => $right];
+                    }
+                }
+                if (!empty($valueConditions)) {
```

Comment:
I'd think that internally `($valueConditions)` is like check the type, compare size against 0, stirred with a shot of black PHP voodoo magic :sparkles: 

I'm fine with any of these variants. `count()` is like 2 to 4 times slower than the other too, in case that is of any concern.